apps/website/src/components/Playground/template/App.js

```diff
@@ -12,12 +12,30 @@ export default function App() {
   const [rules, setRules] = React.useState('');
   const ref = React.useRef(null);
   React.useEffect(() => {
+    const styleTag = document.createElement('style');
+    styleTag.id = 'playground';
```

Comment:
Is this `id` used somehow?

Reply:
not really, it's just for debugging purposes. I can hide it behind a  NODE_ENV check ?

Reply:
If it's not needed I would rather delete it...